src/bios_handler.cpp

```diff
@@ -283,7 +285,15 @@ void IbmBiosHandler::saveAmmToVpd(const std::string& i_memoryMirrorMode)
         (i_memoryMirrorMode == "Enabled" ? constants::AMM_ENABLED_IN_VPD
                                          : constants::AMM_DISABLED_IN_VPD)};
 
-    // TODO: Call write keyword API to update the value in VPD.
+    const auto l_retVal = m_manager->updateKeyword(
```

Comment:
As it is a one time check, you can get rid of "const auto l_retVal" use function call in place and check the return value there itself.